src/main/java/org/zalando/nakadi/domain/EventTypeBase.java

```diff
@@ -63,6 +63,7 @@
     private EventTypeOptions options;
 
     @Valid
+    @Nullable
```

Comment:
no need to have annotation here as it is a private field. `@Nullable` annotation is usually used only for public methods. And getAuthorization() method already has this annotation.